compiler-plugin/src/main/kotlin/arrow/meta/quotes/MetaExtensions.kt

```diff
@@ -199,7 +173,9 @@ fun Meta.importDirective(
   quote(match, map) { ImportDirective(it) }
 
 /**
- * TODO
+ * ```
+ * """ { $`(params)` -> $bodyExpression } """.lambdaExpression
```

Comment:
it's possible for LambdaExpressions to also contain function literals. A weird one I think we would benefit from explaining in examples as mentioned in #130